core/src/test/java/org/infinispan/manager/ClusterExecutorTest.java

```diff
@@ -0,0 +1,327 @@
+package org.infinispan.manager;
+
+import org.infinispan.commons.CacheException;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.test.Exceptions;
+import org.infinispan.test.MultiCacheManagerCallable;
+import org.infinispan.test.TestException;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.infinispan.test.TestingUtil.withCacheManagers;
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+import static org.testng.AssertJUnit.fail;
+
+/**
+ * @author Will Burns
+ * @since 8.2
+ */
+@Test(groups = {"functional", "smoke"}, testName = "manager.CacheManagerTest")
```

Comment:
Updated @wburns! I just overlooked the comment, but clarifying things is never a bad idea ;)